testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -889,29 +893,26 @@
 #
 # Test if an option is selected
 #
-Then(/^option "([^"]*)" is selected as "([^"]*)"$/) do |arg1, arg2|
-  xpath = "//input[@id='#{arg2}']"
-  if has_xpath?(xpath)
-    xpath = "//input[@id='#{arg2}']/../../../../../div/div/div[normalize-space(text())='#{arg1}']"
-    raise "#{arg1} is not selected as #{arg2}" unless has_xpath?(xpath)
-  else
-    raise "#{arg1} is not selected as #{arg2}" unless has_select?(arg2, selected: arg1)
+Then(/^option "([^"]*)" is selected as "([^"]*)"$/) do |option, field|
```

Comment:
It would better be named `should be selected` to follow our conventions.

But maybe not in this PR.

Reply:
Fully agree, the whole sentence doesn't sound well to me.
We can do it in another PR.